### DISCUSSION TITLE
昨天晚上更新了，然后没法用了，是为啥呀，我重下了很多版本都打不开软件！

### DISCUSSION THREAD

**Discussion by lingbing76** [valinet/ExplorerPatcher#2746] [General]
昨天晚上W11更新了，然后ExplorerPatcher没法用了，是为啥呀，我重下了很多版本都打不开软件！

**Comment by pyrates999** [comment]
English please.

**Comment by Amrsatrio** [comment]
This is not the place if you want to rant without explaining what actually happened, what you have tried to stop the crashing so that EP can be used again, your Windows build version, and so on.
